eva-server/src/test/java/uk/ac/ebi/eva/server/repository/VariantEntityRepositoryImplTest.java

```diff
@@ -0,0 +1,172 @@
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.mongodb.MongoDbFactory;
+import org.springframework.data.mongodb.core.convert.MappingMongoConverter;
+import org.springframework.data.mongodb.core.query.Criteria;
+import org.springframework.data.mongodb.core.query.Query;
+import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@EnableMongoRepositories
+@ComponentScan("uk.ac.ebi.eva.server.repository")
+public class VariantEntityRepositoryImplTest {
+
+    @Autowired
+    private ApplicationContext applicationContext;
+
+    @Autowired
+    private MongoDbFactory mongoDbFactory;
+
+    @Autowired
+    private MappingMongoConverter mappingMongoConverter;
+
+    private VariantEntityRepositoryImpl variantEntityRepositoryImpl;
+
+    private Query testQuery;
+    private Query expectedQuery;
+
+    @Before
+    public void setUp() {
+        variantEntityRepositoryImpl = new VariantEntityRepositoryImpl(mongoDbFactory, mappingMongoConverter);
+        testQuery = new Query(Criteria
+                                   .where("chr").is("1")
+                                   .and("start").lte(1).gt(1 - 1000000)
+                                   .and("end").gte(1).lt(1 + 1000000)
+        );
+        expectedQuery = new Query(Criteria
+                                   .where("chr").is("1")
+                                   .and("start").lte(1).gt(1 - 1000000)
+                                   .and("end").gte(1).lt(1 + 1000000)
+        );
+    }
+
+    @Test
+    public void testQueryConsequenceType() throws Exception {
+        List<String> consequenceType = new ArrayList<>();
+        consequenceType.add("SO:0001234");
+        variantEntityRepositoryImpl.queryConsequenceType(testQuery, consequenceType);
+        expectedQuery.addCriteria(Criteria.where("annot.ct.so").in(1234));
+        assertEquals(testQuery, expectedQuery);
+    }
+
+    @Test
+    public void testQueryConsequenceTypeMultiple() throws Exception {
+        List<String> consequenceType = new ArrayList<>();
+        consequenceType.add("SO:0001234");
+        consequenceType.add("SO:0001235");
+        List<Integer> expectedConsequenceType = new ArrayList<>();
+        expectedConsequenceType.add(1234);
+        expectedConsequenceType.add(1235);
+        variantEntityRepositoryImpl.queryConsequenceType(testQuery, consequenceType);
+        expectedQuery.addCriteria(Criteria.where("annot.ct.so").in(expectedConsequenceType));
+        assertEquals(testQuery, expectedQuery);
+    }
+
+    @Test
+    public void testRelationalCriteriaHelperEq() throws Exception {
```

Comment:
I would suggest making this the whole word as in `testRelationalCriteriaHelperEquals`, it is clearer at first sight (applies to the rest of test methods).